src/nvim/state.c

```diff
@@ -124,6 +124,8 @@ void state_handle_k_event(void)
       return;
     }
 
+    // multiqueue_process_events(deferred_autocmd_events);
```

Comment:
Weirdly, we don't need to process our events at all? My testing works fine without it. Is this because `state_handle_k_event` processes all events in the main loop until empty, including children loops?